packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -399,19 +399,41 @@ const defaultConfigs = {
     expression: "10 * DEFI_PULSE_SUSHI_TVL / DEFI_PULSE_UNISWAP_TVL"
   },
   CNYUSD: {
-    type: "tradermade",
-    pair: "CNYUSD",
-    minTimeBetweenUpdates: 600,
-    minuteLookback: 7200,
-    hourlyLookback: 604800,
-    ohlcPeriod: 10 // CNYUSD only available at 10 minute granularity
+    type: "fallback",
+    orderedFeeds: [
+      {
+        type: "tradermade",
+        pair: "CNYUSD",
+        minTimeBetweenUpdates: 600,
+        minuteLookback: 7200,
+        hourlyLookback: 604800,
```

Comment:
Not necessarily suggesting we change it, but why are the default hourly lookbacks for these feeds 7 days? I could see making it 3 hours so we ensure that it at least covers the full 2 hours of the minutely lookback, but it seems like there's no reason the bots would need 7 days? Reducing this may also help with reliability since we're not asking for as much data.

Reply:
Well we want this to at least cover the period of time in which no data is being served over the weekend, so if the `minuteLookback` starts failing (i.e. returns NO data) 2 hours into the weekend (which begins at 22:00 UTC on Friday), then we would want `hourLookback` to cover a bit less than 2 days (but probably more over extended holiday periods). So I think a worst case setting here is 4 days, but I agree 7 days is too much. WDYT

Reply:
Seems reasonable to me!